### PR TITLE
Remove stray periods

### PR DESCRIPTION
**Pull request by jsoref**
### Why:

Random inconsistent periods are yucky

### What's being changed:

old
new

### Check off the following:

- [x] I have reviewed my changes in staging (look for the latest deployment event in your pull request's timeline, then click **View deployment**).
- [ ] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/CONTRIBUTING.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/partner-contributed-documentation.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-name: Partner-owned product documentation.
+name: Partner-owned product documentation
 description: Initiate a set of tasks to be completed by a GitHub partner wishing to document how their product works with GitHub.
 labels:
```
Comment (Realest1Re) [inline .github/ISSUE_TEMPLATE/partner-contributed-documentation.yml:3]: P

**Comment by jsoref** [comment]
Could someone please explain why the review request was removed?

**Review by ramyaparimi — APPROVED** [review]
@jsoref 
Thank you opening a PR with this fix! I'll update the branch and get this merged once tests are passing.

**Comment by Deemanboy** [comment]
Remove

**Pull request merged**